csunplugged/topics/content/en/image-representation/unit-plan/lessons/colour-by-numbers.md

```diff
@@ -0,0 +1,145 @@
+
+- for the first activity choose from the 1 page images (the star, teacup, or the cat).
+  You can use a combination of these images, students do not all need to complete the same one,
+- for the second activity choose from the 6, 8, and 9 page images. Print enough for one set of image worksheets per group of students.
+
+{panel type="general"}
+
+# Notes on resources
+
+- Print spare copies of each worksheet in case students make mistakes and need a new one.
+- Use the **Black and White (2 possible binary values)** colouring option, **not** the option which says "in Run Length Encoding".
+- Students can instead use a black colouring pencil, crayon, or felt tip pen, but easily erasable pencils work much better for the first time students do this activity.
+
+{panel end}
+
+Teacher to class: Computer screens are divided up into a grid of small squares, each of which can display a colour.
+We call these squares picture elements, or pixels.
+
+Write the words "picture elements" on the board.
+To show how we get the word pixels, circle the "pi" of picture and the "el" of elements, and write "pixel" underneath.
+
+Teacher to class: Each of these pixels can be a different colour, and when we have a lot of them on a screen they form an image.
+Has anyone heard of pixels before?
+What about megapixels?
+
+{panel type="general"}
+
+# Teaching observations
+
+Megapixel means one million pixels.
+Students may have heard the term "megapixels" used before because cameras are often described as being a certain number of megapixels, for example a phone might have a *12-megapixel* camera.
+This describes the resolution of the photos a camera can take.
+A 12-mega pixel camera can produce an image with 12 million pixels in it.
+
+Television screens and projectors also display images using pixels.
+
+{panel end}
+
+Teacher to class: Every image we see on a screen, whether it is a picture, a video, or text, is shown using pixels, and all a computer needs to store is what colour each of the pixels on the screen should be.
+
+Show the following images to students on a screen or on the board.
+
+Teacher to class: In a black and white image, each pixel can be either black or white, so all the computer would need to store is which dots are black and which are white.
+For example, if we wanted to display the letter *a* we first need to divide it into squares.
+If we zoom in further and further on the letter we can see a grid of pixels similar to these:
+
+{image file-path="img/topics/letter-zooming-to-pixels.png" alt="Three images of the lower case letter 'a' are shown.
+These progressively zoom in to show the individual black and white squares which make up the letter on the screen."}
+
+We can represent this image using binary digits.
+If a 1 indicates a white square and a 0 indicates a black square then we can represent our letter *a*, on a 5x6 pixel grid, like this:
+
+**10001, 11110, 10000, 01110, 01110, 10000**
```

Comment:
I'd say this is a strange choice of character, because a lot of people don't write their 'a's like this. But that's not for me to decide

Reply:
I choose it because it was what was used in the original unplugged, but I was wondering if it should be changed. A lot of fonts show it like that, but it probably will look odd to a lot of people. I just realised it will look particularly odd to children who are learning how to write!

The images wont take too long to change so I'll try some other characters and find one that I think looks better.

Reply:
I've replaced it with an upper case C (for computer), how does that look? @JackMorganNZ as well.

Reply:
Upper case is good - more immediately recognisable. C might be a bit simple though, I'm not sure